src/Symfony/Component/Yaml/Parser.php

```diff
@@ -178,9 +178,8 @@ private function doParse($value, $flags)
                 $context = 'mapping';
 
                 // force correct settings
-                Inline::parse(null, $flags, $this->refs);
+                Inline::initialize($flags, $this->getRealCurrentLineNb());
```

Comment:
`getRealCurrentLineNb` is also marked as internal. Why not add return type there?